tests/system/common.py

```diff
@@ -771,6 +778,105 @@ def running_task_status(task_statuses):
     assert False, "Did not find a TASK_RUNNING status in task statuses: %s" % (task_statuses,)
 
 
+def unreachable_tasks(framework_name='marathon', app_id=None):
+    """ Returns the unreachable tasks for a given framework.
+    """
+    client = mesos.DCOSClient()
+    frameworks = client.get_master_state()['frameworks']
+    for framework in frameworks:
```

Comment:
we might be able to put this if "if framework["name"] == framework_name:" directly on this line to reduce the nesting?